tests/integration/test_target_duckdb.py

```diff
@@ -1581,3 +1581,39 @@ def test_loading_tables_with_custom_temp_dir(self):
         target_duckdb.persist_lines(self.connection, self.config, tap_lines)
 
         self.assert_multiple_streams_are_into_duckdb()
+
+
+class TestIntegrationLocal:
```

Comment:
I use pytest-style tests because what I'm most familiar with but I can try to change it to be unittest-style.